com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -555,8 +554,8 @@ private void InjectWriteAndCallBlocks(MethodDefinition methodDefinition, CustomA
                         instructions.Add(processor.Create(OpCodes.Ldloc, rpcParamsIdx));
                     }
 
-                    // isReliable
-                    instructions.Add(processor.Create(isReliableRpc ? OpCodes.Ldc_I4_1 : OpCodes.Ldc_I4_0));
+                    // rpcDelivery
+                    instructions.Add(processor.Create(OpCodes.Ldc_I4, (int)rpcDelivery));
```

Comment:
This has to be / should be an int vs. a bool / byte?  I'm not well versed in this...

Reply:
we're putting `int` into IL-side but it inherently converts into the enum type it's referring to. this is valid IL, don't think this being a C# `byte` vs `int` code.